geode-core/src/main/java/org/apache/geode/internal/cache/xmlcache/CacheXmlGenerator.java

```diff
@@ -1253,10 +1253,6 @@ private void generate(Pool cp) throws SAXException {
       }
       if (generateDefaults() || cp.getSocketBufferSize() != PoolFactory.DEFAULT_SOCKET_BUFFER_SIZE)
         atts.addAttribute("", "", SOCKET_BUFFER_SIZE, "", String.valueOf(cp.getSocketBufferSize()));
```

Comment:
Why don't you also remove the code in CacheXmlParser that now calls the deprecated PoolFactory.setThreadLocalConnections? I know you don't need to but it seems cleaner and the product will be that much more ready to have these deprecated apis removed. It would also allow you to delete: CacheXml.THREAD_LOCAL_CONNECTIONS

Reply:
I left it in there so that the warning would get logged of the xml contained this attribute.

Reply:
Good reason